.github/ISSUE_TEMPLATE/1_3_native_image_error_report.yml

```diff
@@ -0,0 +1,59 @@
+name: 🚨 Share Native Image Error Report
```

Comment:
Maybe call this: `Share Native Image Build-Time Crash Report`? @fniephaus

Reply:
We could also use a different icon, e.g., 💥

Reply:
The 🚨 icon indicates a native image issue. I updated a few more icons to match the context of the other components too.

Reply:
Change included. And code updated